src/routes/docs/products/auth/+page.markdoc

```diff
@@ -46,10 +46,13 @@ Implement custom authentication methods like biometric and passkey login by gene
 {% cards_item href="/docs/products/auth/mfa" title="Multifactor authentication (MFA)" %}
 Implementing MFA to add extra layers of security to your app.
 {% /cards_item %}
+{% cards_item href="/docs/products/auth/team-invites" title="Team invites" %}
+Manage team invites and memberships with client-side and server-side approaches.
+{% /cards_item %}
```

Comment:
This isn't an authentication method, though 🧐

Reply:
@stnguyen90 Oh, got it. I've created a section for "Team management". What do you think?

Reply:
It seems odd to have just 1 card. I don't think we need to add anything else about teams/team management on this page.